packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -431,16 +432,25 @@ function i(
 
         if (process.env.NODE_ENV !== 'production') {
             const vnodes = isArray(vnode) ? vnode : [vnode];
-            forEach.call(vnodes, (childVnode: VNode | null) => {
-                if (!isNull(childVnode) && isObject(childVnode) && !isUndefined(childVnode.sel)) {
```

Comment:
`childVnode.sel` is just a sniff test for `VElement` or `VCustomElement` since those are the only types that can have a `sel`. I prefer to be explicit and just check for the `childVnode.type` instead.

Reply:
I wonder if we need the check on `isObject` since `vnodes` [_should_ always contain](https://github.com/salesforce/lwc/pull/4097/files#diff-a284e53a17a2b48cc32e6bf3b121b3acb849a57df886d1cb92a2a46df5aebb7bR426-R431) a `vnode`.

It doesn't break any karma tests when it's removed but I'm not sure if there's any edge cases I'm not thinking of.